sagemaker-featurestore/feature_store_client_side_encryption.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>Once the training example using client sdie encryption is done, it would be fantastic to link it up with the doc on how run a training job with encrypted data (and decrypt it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
So I link this in the last cell, (https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/introduction.html). Is there another link I should also include?